run.py

```diff
@@ -144,21 +139,45 @@ def create_or_update_ecs_service(ecs, ECS_SERVICE_NAME, ECS_TASK_NAME):
 
 def get_queue_url(sqs):
```

Comment:
Why change this function instead of just run it twice, once for the dead letter queue and then once for the regular queue?